<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fort Password Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="FortResources.qrc">
    <normaloff>:/icons/Fort255.png</normaloff>:/icons/Fort255.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLineEdit" name="lineEditSearch">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="placeholderText">
       <string>Type to search</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::SinglePass</enum>
      </property>
      <property name="selectionRectVisible">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLock"/>
    <addaction name="actionExport_As_Plain_Text"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuItem">
    <property name="title">
     <string>Item</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionRemove"/>
    <addaction name="separator"/>
    <addaction name="actionEdit"/>
    <addaction name="actionTag"/>
   </widget>
   <widget class="QMenu" name="menuManage">
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="actionMaster_Passphrase"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuItem"/>
   <addaction name="menuManage"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLock"/>
   <addaction name="separator"/>
   <addaction name="actionNew"/>
   <addaction name="actionRemove"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="separator"/>
   <addaction name="actionTag"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="FortResources.qrc">
     <normaloff>:/icons/DocumentNew.png</normaloff>:/icons/DocumentNew.png</iconset>
   </property>
   <property name="text">
    <string>Add...</string>
   </property>
   <property name="toolTip">
    <string>Add new item</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="FortResources.qrc">
     <normaloff>:/icons/DocumentDelete.png</normaloff>:/icons/DocumentDelete.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove selected item</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="FortResources.qrc">
     <normaloff>:/icons/Clipboard.png</normaloff>:/icons/Clipboard.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy selected item password to clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionLock">
   <property name="icon">
    <iconset resource="FortResources.qrc">
     <normaloff>:/icons/Keychain.png</normaloff>:/icons/Keychain.png</iconset>
   </property>
   <property name="text">
    <string>Lock</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit...</string>
   </property>
   <property name="toolTip">
    <string>Edit selected item</string>
   </property>
  </action>
  <action name="actionOpen_url">
   <property name="text">
    <string>Open url...</string>
   </property>
   <property name="toolTip">
    <string>Navigate to url</string>
   </property>
  </action>
  <action name="actionTag">
   <property name="icon">
    <iconset resource="FortResources.qrc">
     <normaloff>:/icons/Tag.png</normaloff>:/icons/Tag.png</iconset>
   </property>
   <property name="text">
    <string>Tag As Favorite</string>
   </property>
   <property name="iconText">
    <string>Tag</string>
   </property>
   <property name="toolTip">
    <string>Tag as favorite</string>
   </property>
  </action>
  <action name="actionMaster_Passphrase">
   <property name="text">
    <string>Master Passphrase...</string>
   </property>
  </action>
  <action name="actionExport_As_Plain_Text">
   <property name="text">
    <string>Export As Plain Text...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FortResources.qrc"/>
 </resources>
 <connections/>
</ui>
